bigbluebutton-html5/imports/ui/components/presentation/component.jsx

```diff
@@ -231,7 +243,10 @@ class PresentationArea extends Component {
     return (
       <div
         style={svgDimensions}
+        aria-label="Slide Content"
+        aria-describedby="currentSlideText"
       >
+        <p id="currentSlideText" hidden>{slideContent}</p>
```

Comment:
Using `hidden` like this also hides the content from NVDA, I'd suggest adding a `className` to hide it visually.
https://github.com/bigbluebutton/bigbluebutton/blob/master/bigbluebutton-html5/imports/ui/components/nav-bar/recording-indicator/styles.scss#L4

Reply:
Done in last commit for this PR.